### ISSUE TITLE
No Output

### ISSUE THREAD

**Issue by Gwinert** [th317erd/gh-wiki-gen#1]
I ran this library in the repo directory, and it has the correct Input and Output path displayed, however, no changes are made in my .wiki folder. Am I doing something wrong? Or is the Library looking for something that I'm not providing?

**Comment by th317erd** [comment]
Gwinert, I am currently at railsconf so am unable to dive in and help you
at the moment, but I would be more than happy to assist you and help you
work through your problem as I am able. Can you reply with some of your
configuration / code?

**Comment by Gwinert** [comment]
I was just testing it on an example file. 
my docGen.js looks like this:
```
//docGen.js
module.exports = function() {
    return {
        //RegExp include / exclude / extensions (extensions are smart) 
        include: ['^src/.*'],
        exclude: [],
        extensions: ['js'],
        inputPath: './',
        outputPath: '../Example-Project.wiki'
    };
};
```

and my simple JS file looks like
```
/** This is a description of the foo function. */
function foo() {
	return bar;
}

/**
 * Represents a book.
 
 * @constructor
 * @param {string} {title} - The title of the book.
 * @param {string} {author} - The author of the book.
 */
function book(title, author) {

}
```
Let me know if you any other files/code should be referenced in order for you to help me understand the issue. 
Thanks!

**Comment by th317erd** [comment]
My guess without being able to check is that you have no root specified
(namespace, or class). You can download the module "devoir" from npm and
look at its source code to get a better idea of what might be wrong. You
could also try putting a "@namespace test" comment at the top of your
example code and see if that helps. I will get you more information on your
example when I have an opportunity to play with it myself.

**Comment by th317erd** [comment]
Okay, this is working (better):

/**
@namespace {test} This is a description
*/

/**
@function {foo} This is a description of the foo function.
@return {String} Return a string
*/
function foo() {
  return bar;
}

/**
 * @function {book} Represents a book.
 * @param {string} {title} - The title of the book.
 * @param {string} {author} - The author of the book.
 */
function book(title, author) {

}

It still has some issues. This project is still ALPHA, and has some bugs (I
have an issue in one of my projects where it gets stuck in an infinite
recursive loop). However, if you still want to use it I will help you along
the way as I can. I just haven't had time to work on it, and didn't think
anyone was using it so I haven't given it priority.

Also, FYI, some of the tags have been changed from what is on the NPM
documentation (that I haven't had a chance to update). I just pushed an
update to npm that might help you out a bit. Let me know if you have any
further issues or questions.

~th317erd

**Comment by Gwinert** [comment]
Thanks for taking a look. So I tried making my documentation look like yours does, and then I tried running the command again. 
While it is still getting the correct Input Path and Output path, it's unfortunately not generating anyfiles in the repo.wiki.
To be clear, I just go to my repo where I have my JSDocumented files, and then I run the command "gh-wiki-gen" from that directory, then it should generate a .MD file in my ../repo.wiki directory? Let me know if that is clear, I could also be potentially missing a step.

Thank you for taking time to clarify the issue.

**Comment by th317erd** [comment]
Yes, but it will generate multiple md files and also a "docs.json". If it
isn't outputting anything at all then you probably have an issue with your
path, or permissions on the filesystem. What version of node are you using?

**Comment by Gwinert** [comment]
My path appears to be pointing to the correct directory, and I'm using 7.9.0, the most recent. Other Libraries with similar functionality haven't had a problem with writing files. but I'll fiddle around with it.

**Comment by th317erd** [comment]
Well, since the code is in alpha and I have been working on it, it actually
dumps out a lot of information, including when it writes a file and where.
You should see it telling you where it is writing files when you run it. If
you don't see any of that, maybe there is still a problem

**Comment by Gwinert** [comment]
Thanks for all the quick responses. So I should be seeing more than just this in my Command line when I run it?
[Command Line Image](http://pasteboard.co/8Z7pP8QcQ.png)

**Comment by th317erd** [comment]
Yes, absolutely. Something else is going wrong. It looks like it didn't
even parse your files. I have never attempted to run it on windows (though
in theory it should work just fine). Maybe you are running into a cross
platform issue? I will have to try and run it on windows myself when I have
time. I might be able to look at it tonight. In the meantime, if you want
to, you might consider throwing a few debugging logs throughout different
parts of the code to see where it is bailing
